src/GenerateConstraints.hs

```diff
@@ -89,8 +89,7 @@ genConstraints globalEnv root rootSig = fmap sort (gen root)
                                 trueType <- toEither (ty ifTrue) (ExpressionMissingType ifTrue)
                                 falseType <- toEither (ty ifFalse) (ExpressionMissingType ifFalse)
                                 let expected = XObj (Sym (SymPath [] "Condition in if value") Symbol) (info expr) (Just BoolTy)
-                                let lol = XObj (Sym (SymPath [] "lol") Symbol) (info expr) (Just BoolTy)
```

Comment:
heh, I always wondered about this :)

Reply:
🤔

Reply:
Probably some test by @hellerve, introduced in 6e147598a.